client/my-sites/media-library/index.jsx

```diff
@@ -20,16 +19,17 @@ import { filterItemsByMimePrefix } from 'calypso/lib/media/utils';
 import filterToMimePrefix from './filter-to-mime-prefix';
 import FilterBar from './filter-bar';
 import QueryPreferences from 'calypso/components/data/query-preferences';
+import QuerySiteFeatures from 'calypso/components/data/query-site-features';
```

Comment:
I've mentioned this in another PR, but if the features information is also returned from the main `/site` endpoint, then could we use that and remove the need to query for it separately?

If we do want to do that, it might be best to handle the refactor in a separate PR.

Reply:
yeah, that could be a nice idea. I decided initially to get the features in a separate request to don't load the initial process of pulling sites data too much. However, it's something that we can still consider as a valid enhancement.